webui/src/components/editrecord.jsx

```diff
@@ -465,13 +469,21 @@ const EditRecord = React.createClass({
             }
             record = addEmptyMetadataBlocks(record, props.blockSchemas) || record;
             this.setState({record});
+            this.setState({checkboxValue:record.get('publication_state')});
         } else if (this.state.record && props.blockSchemas) {
             const record = addEmptyMetadataBlocks(this.state.record, props.blockSchemas);
             if (record) {
                 this.setState({record});
             }
         }
 
+        // Record is submitted for review: the publication_state is 'submitted' and readonly should be True
+        if(this.props.community && this.state.record){
+            if(this.props.community.getIn(["publication_workflow"]) == 'review_and_publish' && this.state.record.get('publication_state')=='submitted' && !this.state.revoking){
```

Comment:
`this.props.community.getIn(["publication_workflow"])` -> `this.props.community.get("publication_workflow")`